editor/blocks/quote/index.js

```diff
@@ -0,0 +1,44 @@
+const Editable = wp.blocks.Editable;
+const { query, html } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'core/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+	category: 'quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: html( 'cite,footer' )
+	},
+
+	edit( attributes, onChange ) {
+		const { value, citation } = attributes;
+
+		// Concat strings parsed by hpq from paragraph values into one string.
+		const combinedValue = value.join( '' );
+		return (
+			<blockquote>
+				<Editable
+					value={ combinedValue }
+					onChange={ ( newValue ) => onChange( { newValue } ) } />
+				<cite>
+					<Editable
+						value={ citation }
+						onChange={ ( newValue ) => onChange( { newValue } ) } />
```

Comment:
I think it is `{ value: newValue }` again?  `value` is a prop for the `Editable` component, where as `citation` is a prop for the quote component.  onChange is feeding the `Editable` component.